src/components/NavigationPanel/index.jsx

```diff
@@ -14,9 +21,13 @@ class NavigationPanel extends React.Component {
   }
 
   closeNavWhenShrunk = () => {
-    if (window.matchMedia('(max-width: 992px)').matches && this.props.navPanelOpen) {
+    const widthThreshold = 1240;
```

Comment:
is there a way to get this threshold dynamically? It's the current breakpoint for mobile but may change in the future.